crates/pipeline-manager/Cargo.toml

```diff
@@ -15,86 +15,86 @@ development = ["pg-client-config"]
 
 [dependencies]
 # Feldera
-feldera-types = { path = "../feldera-types" }
+feldera-types = { workspace = true }
 
 # Cryptography provider
 # Make sure this is the same rustls version used by other crates in the dependency tree.
 # See the `ensure_default_crypto_provider` function at the root of this crate.
-rustls = "0.23.12"
+rustls = { workspace = true }
 
 # Logging
-log = "0.4.20"
-env_logger = "0.10.0"
-colored = "2.0.0"
+log = { workspace = true }
+env_logger = { workspace = true, features = ["color"] }
+colored = { workspace = true }
 
 # Error handling and asserts
-anyhow = { version = "1.0.57", features = ["backtrace"] }
-thiserror = "1.0"
-static_assertions = "1.1.0"
+anyhow = { workspace = true, features = ["backtrace"] }
+thiserror = { workspace = true }
+static_assertions = { workspace = true }
 
 # Asynchronous runtime
-tokio = { version = "1.25.0", features = ["rt-multi-thread", "fs", "macros", "process", "io-util", "io-std", "signal"] }
-tokio-stream = { version = "0.1.15" }
-async-stream = { version = "0.3.5" }
-async-trait = "0.1"
-futures-util = "0.3.30"
+tokio = { workspace = true, features = ["rt-multi-thread", "fs", "macros", "process", "io-util", "io-std", "signal"] }
+tokio-stream = { workspace = true }
+async-stream = { workspace = true }
+async-trait = { workspace = true }
+futures-util = { workspace = true }
 
 # Encoding, decoding and formatting
-rand = "0.8.5"
-uuid = { version = "1.11.0", features = ["v7", "std", "serde"] }
-chrono = { version = "0.4.38", default-features = false, features = ["clock", "serde"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = { version = "1.0.127", features = ["preserve_order"] }
-serde_yaml = "0.9.34"
-hex = "0.4.3"
-indoc = "2.0.5"
-openssl = "0.10.66"
-url = {version = "2.4.0"}
-urlencoding = "2.1.3"
-regex = "1.10.2"
-tar = "0.4.44"
-flate2 = "1.1.0"
-base64 = "0.22.1"
+rand = { workspace = true }
+uuid = { workspace = true, features = ["v7", "std", "serde"] }
+chrono = { workspace = true, features = ["clock", "serde"] }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true, features = ["preserve_order"] }
+serde_yaml = { workspace = true }
+hex = { workspace = true }
+indoc = { workspace = true }
+openssl = { workspace = true }
+url = { workspace = true }
+urlencoding = { workspace = true }
+regex = { workspace = true }
+tar = { workspace = true }
+flate2 = { workspace = true }
+base64 = { workspace = true }
 
 # Command-line interaction
 clap = { version = "4.0.32", features = ["derive", "env"] }
-termbg = "0.5.1"
+termbg = { workspace = true }
 
 # Database
-tokio-postgres = { version = "0.7", features = ["with-serde_json-1", "with-uuid-1", "with-chrono-0_4"]}
-deadpool-postgres = "0.10.5"
+tokio-postgres = { workspace = true, features = ["with-serde_json-1", "with-uuid-1", "with-chrono-0_4"] }
+deadpool-postgres = { workspace = true }
 # Waiting for https://github.com/faokunega/pg-embed/pull/26
```

Comment:
This is now merged, is it still necessary to have this be a git dependency?

Reply:
will check and fix in another PR if it works now